### ISSUE TITLE
Google Earth : set initial view

### ISSUE THREAD

**Issue by donnacalhoun** [clawpack/visclaw#126]
Set initial range (distance from earth), camera tilt angle as option in setplot.py.   Same as : 

```Python
    plotfigure.show = True   # Don't show this file in the html version
    plotfigure.use_for_kml = True
    plotfigure.kml_dpi = 500
    plotfigure.kml_tilt = 45       # For example
    plotfigure.kml_xlimits = [-120,-60]
    plotfigure.kml_ylimits = [-60, 0.0];
```

**Comment by donnacalhoun** [comment]
this is mostly fixed.  One remaining issue is the initial height.  While the current seeing will work for large scale events (Chile 2010 tsunami), the user must manually zoom in in GE to see smaller scale simulations (LaPush, for example).

**Comment by donnacalhoun** [comment]
I now set the initial height to twice the width across the computational domain.  This seems to give a reasonable initial view from the examples I have tried.    Since the user can have multiple figures in a KML file, so to specify which one should be used for an initial view, they would set : 

    plotfigure.kml_use_for_initial_view = True. 

By default (or if this is not set), the first figure found will be used for the initial view.

I'd say this issue is closed ...
